### ISSUE TITLE
unable to use `shift shift` as a shortcut

### ISSUE THREAD

**Issue by Yneth** [zed-industries/community#1787]
### Check for existing issues

- [X] Completed

### Describe the feature

As a user I want the Zed editor to support the `shift shift` shortcut to start searching everywhere.
Not particularly interested in the search everywhere functionality. Mainly interested in shortcut to work.

Expected behavior:
`shift shift` performs assigned shortcut.

Actual behavior:
nothing happens. All shortcuts defined after the definition of `shift shift` stop working

**Comment by JosephTLyons** [comment]
Yeah, you're right - we don't have the ability to assign something like that just yet. Just curious - are you coming from a JetBrains IDE? I'm assuming that's where the convention of `shift-shift` is coming from.

**Comment by Yneth** [comment]
@JosephTLyons  yes, I wanted to migrate my shortcuts from JetBrains IDE
